From 23ddd759ec9fdfbb3041c95123f6309fb10636e6 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:32:33 -0400
Subject: [alsa] fm801: convert struct description to kernel-doc

Message-id: <140972823573-299-patchset-jkysela@redhat.com>
Patchwork-id: 90995
O-Subject: [RHEL 7.1 PATCH 298/442] ALSA: fm801: convert struct description to kernel-doc
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Just move field descriptions to the struct description in the kernel-doc
format. There is no functional change.

Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 052c233e9800b1f7da4c18cb1660576249486a73)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/fm801.c b/sound/pci/fm801.c
index fb7141b..11d9d6f 100644
--- a/sound/pci/fm801.c
+++ b/sound/pci/fm801.c
@@ -158,21 +158,27 @@ MODULE_PARM_DESC(radio_nr, "Radio device numbers");
 #define FM801_GPIO_GS3  (1<<15)
 #define FM801_GPIO_GS(x) (1<<(12+(x)))
  
-/*
-
+/**
+ * struct fm801 - describes FM801 chip
+ * @port:  I/O port number
+ * @multichannel: multichannel support
+ * @secondary:  secondary codec
+ * @secondary_addr: address of the secondary codec
+ * @tea575x_tuner: tuner access method & flags
+ * @ply_ctrl:  playback control
+ * @cap_ctrl:  capture control
  */
-
 struct fm801 {
  int irq;
 
- unsigned long port; /* I/O port number */
- unsigned int multichannel: 1, /* multichannel support */
-       secondary: 1; /* secondary codec */
- unsigned char secondary_addr; /* address of the secondary codec */
- unsigned int tea575x_tuner; /* tuner access method & flags */
+ unsigned long port;
+ unsigned int multichannel: 1,
+       secondary: 1;
+ unsigned char secondary_addr;
+ unsigned int tea575x_tuner;
 
- unsigned short ply_ctrl; /* playback control */
- unsigned short cap_ctrl; /* capture control */
+ unsigned short ply_ctrl;
+ unsigned short cap_ctrl;
 
  unsigned long ply_buffer;
  unsigned int ply_buf;
-- 
1.7.1